exercises/node-javascript/multi-cmk-complete/retrieve.js

```diff
@@ -0,0 +1,23 @@
+const { decryptStream, KmsKeyringNode } = require("@aws-crypto/client-node");
```

Comment:
@mattsb42-aws @lizroth Here is the complete retrieve file.  Both CMKs have been added.